src/hooks/use-site-details.tsx

```diff
@@ -295,6 +295,15 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 						),
 						showOpenLogs: true,
 					} );
+				} else if ( error instanceof Error && error.message.includes( 'ERROR_PORT_IN_USE' ) ) {
+					const port = error.message.match( /\d+/ );
+					getIpcApi().showErrorMessageBox( {
+						title: __( 'Failed to start the site server' ),
+						message: __(
+							`The site server failed to start because the port is already in use. Please close any local development apps that may be using port ${ port } and try again.`
+						),
+						showOpenLogs: true,
```

Comment:
Will logs be helpful in this case?

Reply:
Not really. I was skeptical about removing it. I will go ahead and remove it.

Reply:
Addressed in [a22a794](https://github.com/Automattic/studio/pull/1236/commits/a22a794e34cae0e9579bcd12e72931b3bcaf772b)